src/frontend/src/common/components/library/data_subview/components/RedirectConfirmationModal/style.ts

```diff
@@ -4,16 +4,10 @@ import { P } from "src/common/styles/support/style";
 
 export const StyledHeader = styled.div`
   ${fontHeaderXl}
-`;
-
-export const StyledImg = styled("img")`
-  height: 45px;
-
   ${(props) => {
     const spaces = getSpaces(props);
-
     return `
-      margin-bottom: ${spaces?.l}px;
+      margin-top: ${spaces?.l}px;
```

Comment:
had to add margin-top to `StyledHeader`, styling wasn't being properly applied to Next Image component and this seemed like a more straightforwards work around